common/alert/reports/a11y-1.json

```
@@ -0,0 +1,908 @@
+    },
+    {
+      "@id": "_:sc_1.4.1",
+      "earl:test": "WCAG21:use-of-color",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "Pass as long as title, or paragraph indicate the type of alert; especially for <p> inside div",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G14",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.2",
+      "earl:test": "WCAG21:audio-control",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:result": [],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.3",
+      "earl:test": "WCAG21:contrast-minimum",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G18",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.4",
+      "earl:test": "WCAG21:resize-text",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G142",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.5",
+      "earl:test": "WCAG21:image-of-text",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "I have mentioned C22 because it mentioned about using css classes",
+      "earl:result": ["WCAG21Tech:general/C22"],
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:css/C22",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.10",
+      "earl:test": "WCAG21:reflow",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "All other success criterias are checking for 400% zoom level, which is addressed by C32",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:css/C32",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
```

To fix the CI errors about JSON validation
```suggestion
      "earl:result": [
        {
            "earl:test": "WCAG2xTech:css/C32",
            "earl:outcome": "earl:passed",
            "earl:subject": "_:subject"
        }
      ],
      "earl:mode": "earl:manual",
      "@type": ["earl:TestResult", "earl:Assertion"]
```